applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -216,6 +228,94 @@ load_conference(DocId, Context) ->
+        {_, JObjs} -> participant_details(conference_participants(JObjs), Context)
+    end.
+
+-spec conference_participants(wh_json:objects()) -> wh_json:objects().
+conference_participants(JObjs) ->
+    [Participant || JObj <- JObjs
+                    ,Participant <- wh_json:get_value(<<"Participants">>, JObj, [])
+    ].
+
+-spec participant_details(wh_json:objects(), cb_context:context()) -> cb_context:context().
+-spec participant_details(wh_json:objects(), cb_context:context(), wh_json:objects()) -> cb_context:context().
+participant_details([], Context) ->
+    cb_context:add_system_error('conference_not_active', Context);
+participant_details(Participants, Context) ->
+    participant_details(Participants, Context, []).
+
+participant_details([], Context, Acc) ->
+    crossbar_doc:handle_json_success(Acc, Context);
+participant_details([H|T], Context, Acc) ->
+    CallId = wh_json:get_value(<<"Call-ID">>, H),
+    Req = [{<<"Call-ID">>, CallId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    case whapps_util:amqp_pool_request(Req
+                                       ,fun wapi_call:publish_channel_status_req/1
+                                       ,fun wapi_call:channel_status_resp_v/1
+                                      ) of
+        {'error', E} ->
+            lager:debug("error fetching channel status for ~s (~p)", [CallId, E]),
+            participant_details(T, Context, Acc);
+        {'ok', Resp} ->
+            Participant = wh_json:set_values([{<<"Participant-ID">>, wh_json:get_ne_binary_value(<<"Participant-ID">>, H)}
+                                              ,{<<"Mute">>, not wh_json:is_true(<<"Speak">>, H)}
+                                             ], Resp),
+            participant_details(T, Context, [Participant | Acc])
+    end.
+
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%% Perform conference kick/mute/unmute via API
+%% @end
+%%--------------------------------------------------------------------
+-spec do_conference_action(cb_context:context(), path_token(), wh_json:json_term(), wh_json:json_term()) ->
+                             cb_context:context().
+do_conference_action(Context, _, _, 'undefined') ->
+    cb_context:add_system_error('participant_missing', Context);
+do_conference_action(Context, Id, Action, ParticipantId) ->
+    AuthDoc = cb_context:auth_doc(Context),
+    ConfDoc = cb_context:doc(load_conference(Id, Context)),
+    ConfOwnerId = wh_json:get_value(<<"owner_id">>, ConfDoc),
+    AuthOwnerId = wh_json:get_value(<<"owner_id">>, AuthDoc),
+    % Abort if the user is not room owner
+    Resp = case ConfOwnerId of
+        AuthOwnerId ->
+            Conference = whapps_conference:set_id(Id, whapps_conference:new()),
+            perform_conference_action(Conference, Action, ParticipantId);
+        _ -> <<"user_not_owner">>
```

Comment:
@danielfinke 

This should not be a successful response ;)

Could you update it to use:
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/cb_context.erl#L983
and only successful from the above case clause?